wiki/versions/1.19/entity/entity_attributes/groovy/src/main/groovy/org/quiltmc/wiki/entity_attributes/AttributesExample.groovy

```diff
@@ -0,0 +1,45 @@
+package org.quiltmc.wiki.entity_attributes
+
+import net.minecraft.entity.attribute.ClampedEntityAttribute
+import net.minecraft.entity.attribute.EntityAttribute
+import net.minecraft.entity.attribute.EntityAttributeModifier
+import net.minecraft.item.ItemGroup
+import net.minecraft.util.Identifier
+import net.minecraft.util.registry.Registry
+import org.quiltmc.loader.api.ModContainer
+import org.quiltmc.qsl.base.api.entrypoint.ModInitializer
+import org.quiltmc.qsl.command.api.CommandRegistrationCallback
+import org.quiltmc.qsl.item.setting.api.QuiltItemSettings
+
+class AttributesExample implements ModInitializer {
+
+    // @start Attribute-Instance
+    static final EntityAttribute GENERIC_JUMP_BOOST = new ClampedEntityAttribute("attribute.name.generic_jump_boost", 1.0, 0.0, 2.0).setTracked(true)
+    // @end Attribute-Instance
+
+    // @start Modifier-Instance
+    static final UUID SOME_MODIFIER_ID = UUID.fromString("de17612a-adec-11ec-b909-0242ac120002")
```

Comment:
Please use single quotes for strings in groovy - double quotes constructs a GString, which is generally unnecessary unless you're doing `"${}"` string substitution, and can lead to weird behavior in some contexts